README.md

```diff
@@ -0,0 +1,108 @@
+# pyobis
+
+[![pypi](https://img.shields.io/pypi/v/pyobis.svg)](https://pypi.python.org/pypi/pyobis)
+[![docs](https://github.com/iobis/pyobis/actions/workflows/deploy-docs.yml/badge.svg)](https://iobis.github.ic/pyobis)
+[![tests](https://github.com/iobis/pyobis/actions/workflows/tests.yml/badge.svg)](https://github.com/iobis/pyobis/actions/workflows/tests.yml)
+
+Python client for the `OBIS API(https://api.obis.org/).
+
+[Source on GitHub at iobis/pyobis](https://github.com/iobis/pyobis)
+
+## What is it?
+Pyobis is an interesting python package that helps users fetch data from OBIS API which
+holds a great amount of ocean open-data, with ease.
+
+[The Ocean Biodiversity Information System (OBIS)](https://obis.org) a global open-access data and
+information clearing-house on marine biodiversity for science, conservation, and sustainable
+development, maintained by IOOS, harvests occurrence records from thousands of datasets
+and makes them available as a single integrated dataset via various services including the
+OBIS API.
+
+Other OBIS clients:
+
++ R: `robis`, [iobis/robis](https://github.com/iobis/robis)
```

Comment:
This is kind of duplicated, no? Maybe keep only the one after the comma.

Reply:
Thank you for the review! I have updated the README accordingly.